src/rgw/rgw_rest.cc

```diff
@@ -759,6 +760,20 @@ int dump_body(struct req_state* const s,
               const char* const buf,
               const size_t len)
 {
+  bool healthchk = false;
+  // we dont want to limit health checks
+  if(s->op_type == RGW_OP_GET_HEALTH_CHECK)
+    healthchk = true;
+  if(len > 0 && !healthchk) {
+    std::string userfind;
+    s->user->get_id().to_str(userfind);
+    userfind = "u" + userfind;
```

Comment:
There's a lot of these open-coded string setups for getting keys.  Maybe a pair of helper functions would make this cleaner, and allow us to change if necessary in the future?